ios/FluentUI/Core/Theme/FluentColor.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// Wrapper view that returns the appropriate `Color` for a given `ColorSet` in a given environment.
+struct FluentColor: View {
```

Comment:
Are you planning on having something similar for UIColor too? (the conversion from colorset to UIColor)

Reply:
Yep! Should be a pretty simple add-on in the UIColor extension file. In fact I'll write it now while I'm thinking of it.